Makefile-libostree.am

```diff
@@ -156,7 +156,15 @@ libostree_1_la_LIBADD = libotutil.la libglnx.la libbsdiff.la libostree-kernel-ar
 libostree_1_la_LIBADD += $(bupsplitpath)
 EXTRA_libostree_1_la_DEPENDENCIES = $(top_srcdir)/src/libostree/libostree.sym
 
-EXTRA_DIST += src/libostree/libostree.sym
+EXTRA_DIST += \
+	src/libostree/libostree.sym \
+	src/libostree/libostree-experimental.sym \
```

Comment:
I'm ok with this as is...but thinking 🤔 about it more - how about adding a `ostree-exp.pc`/`libostree-exp.so`?  If we were good about bumping the soname every time we changed API, I think I could use it for rpm-ostree too in Fedora.

Did we discuss the "separate .so" option?  I don't recall we did.